### PR TITLE
fundamentals diagnostic response

### PR DESCRIPTION
**Pull request by lcurran**
Lindsey Curran 5/16/2016

### CODE REVIEW

**Comment by lcurran** [comment]
fundamentals diagnostic

File: diagnostic.md [hunk 1]
```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/users/grace/wdi/try
```
Comment (berziiii) [inline diagnostic.md:19]: Correct.

File: diagnostic.md [hunk 2]
```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
...
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+It's an online git repository hosting service
```
Comment (berziiii) [inline diagnostic.md:25]: Correct. But it is also a User Interface for use to interact with Git and Version Control.

File: diagnostic.md [hunk 3]
```diff
@@ -32,7 +32,8 @@ _Replace this text with your answer._
 
 Is the value of this expression `true` or `false`?  Explain your answer.
 
-_Replace this text with your answer._
+'||' expression evaluates to true, '&&' side evaluates to false, therefore
```
Comment (berziiii) [inline diagnostic.md:35]: Correct.

File: diagnostic.md [hunk 4]
```diff
@@ -45,7 +46,7 @@ while (x < 5) {
 
 How many times does the loop run?  What's the value of `x` after the while loop?
 
-_Replace this text with your answer._
+Loops 3 times, final value of x=8
```
Comment (berziiii) [inline diagnostic.md:49]: Correct.

File: diagnostic.md [hunk 5]
```diff
@@ -62,7 +63,7 @@ var result = strangeAdd(0, -5);
 
 What is the value of result?  Explain your answer.
 
-_Replace this text with your answer._
+result = 5. b is less than 0, so we return 0 -(-5).
```
Comment (berziiii) [inline diagnostic.md:66]: Correct.

File: diagnostic.md [hunk 6]
```diff
@@ -76,7 +77,7 @@ var groceryList = [
 
 What is the value of `groceryList[2]`?
 
-_Replace this text with your answer._
+'a stick of butter'
```
Comment (berziiii) [inline diagnostic.md:80]: Correct.

File: diagnostic.md [hunk 7]
```diff
@@ -94,4 +95,4 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+1) 'exercise' 2) { given: 'Jason', surname: 'Weeks'} 3) 'Weeks'
```
Comment (berziiii) [inline diagnostic.md:98]: Correct.

**Comment by berziiii** [comment]
9 correct out of 9 possible. Great start!

**Pull request closed without merging**